src/ui/screens/settings/privacy/ViewPrivacy.qml

```diff
@@ -16,6 +16,25 @@ MZViewBase {
     id: root
     objectName: "privacySettingsView"
 
+    property Component rightMenuButton: Component {
+        Loader {
+            active: MZFeatureList.get("helpSheets").isSupported
+            sourceComponent: MZIconButton {
+                onClicked: helpSheetLoader.active = true
+
+                accessibleName: MZI18n.GlobalHelp
+                Accessible.ignored: !visible
```

Comment:
MZButtonBase has `Accessible.ignored: !visible` this already. Same comment for related PRs.